include/aikido/robot/util.hpp

```diff
@@ -29,7 +29,7 @@ namespace util {
 const std::vector<double> defaultVFParams{
     0.03,  // distanceTolerance
     0.004, // positionTolerance
-    0.004, // angularTolerance
+    0.01,  // angularTolerance
```

Comment:
Just flagging that this change to default params got included in the push, in case it was unintentional

Reply:
Yeah, the old tolerance was too strict for JACO 2 (it was good for Gen3)